src/client_config.rs

```diff
@@ -130,6 +136,7 @@ impl ClientConfig {
                     println!("  -u, --upload\t\t\t\tSets the client to upload mode, Ignores all previous download flags");
                     println!("  -d, --download\t\t\tSet the client to download mode, Invalidates all previous upload flags");
                     println!("  -rd, --receive-directory <DIRECTORY>\tSet the directory to receive files when in Download mode (default: current working directory)");
+                    println!("  --dont-clean\t\t\t\tWill prevent client from deleting files after receiving errors.");
```

Comment:
Can we make the help text "Prevent client from deleting files after receiving errors" (no `.`, no `Will`) in order for it to be more consistent with the remainder?

Reply:
Yes of course. I spent 15 minutes trying to came up with description and not making bool named "dont-clean"

Reply:
haha, maybe something like `--keep-on-error` is more descriptive? What do you think?